modules/agar/src/main/ts/ephox/agar/api/Mouse.ts

```diff
@@ -14,6 +14,8 @@ const mouseMove = (element: SugarElement<Node>, settings: Clicks.Settings = { })
 const mouseOut = (element: SugarElement<Node>, settings: Clicks.Settings = { }): void => Clicks.mouseOut(settings)(element);
 const mouseMoveTo = (element: SugarElement<Node>, dx: number, dy: number, settings: Omit<Clicks.Settings, 'dx' | 'dy'> = { }): void =>
   Clicks.mouseMove({ ...settings, dx, dy })(element);
+const mouseUpTo = (element: SugarElement<Node>, dx: number, dy: number, settings: Omit<Clicks.Settings, 'dx' | 'dy'> = { }): void =>
```

Comment:
How was this working before without this function?

Reply:
It was using `cMouseUpTo` so it was just something I missed when converting the chain/step functions to something that we could us in BDD tests.